core/src/rpc.rs

```diff
@@ -813,6 +834,18 @@ impl RpcSol for RpcSolImpl {
             .get_recent_blockhash(commitment)
     }
 
+    fn get_recent_fee_rate_governor(
+        &self,
+        meta: Self::Metadata,
+        commitment: Option<CommitmentConfig>,
```

Comment:
Adding a `CommitmentConfig` option also seems a little much given that the governor will likely change on rare epoch boundary.  Just returning the governor from the last root seems totally sufficient